### ISSUE TITLE
Transferred project (official organization)

### ISSUE THREAD

**Issue by LordStephen77** [LordStephen77/DreamBlocks-Beyond-of-Time#14]
I've created an organization on the project github, which makes the job more tidy and easy to handle.

## Link organization:

https://github.com/DreamBlocks/DreamBlocks

**Comment by LordStephen77** [comment]
@AKryukov92 @azekillDIABLO @tobyplowy @Gerold55 I also published the official site that did azekillDIABLO together with all of you. You work on the site (azekillDIABLO, tobyplowy, Gerold55), in the organization's repository I put the link to the executable .jar executable (thanks AKryukov92 for your wonderful work on the code).

Add AKryukov92 in the credits of the website. Improve fonts for better reading and insert all new links (organization, jar executable). Thank you all. The project will grow over time and will improve if we work together passionately.

I love you all <3

**Comment by LordStephen77** [comment]
@AKryukov92 @azekillDIABLO @tobyplowy @Gerold55 Please give me confirmation that you have all read. This repository will be deleted, we will only use those in the organization, we must work there. Thank you :)

**Comment by tobyplowy** [comment]
@LordStephen77 yes i have

**Comment by AKryukov92** [comment]
@LordStephen77 I read your message and joined your organization. Probably too soon. Depends on whether this project be public and free or not.

**Comment by LordStephen77** [comment]
@AKryukov92 Yes, the project is open source under license GPL 3.0 is everyone! For this I wanted to create the official organization. DreamBlocks is totally open. You can feel comfortable. Thank you.

Within a few hours to gate this repository, the project is in the official organization, we work there. This repository no longer needs, so the project is even more open! Thank you all.
